docs/upgrade.html

```diff
@@ -80,10 +80,11 @@
     <li> Similarly, when compressing data with gzip, the producer and broker will use 8 KB instead of 1 KB as the buffer size. The default
          for gzip is excessively low (512 bytes). </li>
     <li>The broker configuration <code>max.message.bytes</code> now applies to the total size of a batch of messages.
-        Previously the setting applied to batches of compressed messages, or to non-compressed messages individually. In practice,
-        the change is minor since a message batch may consist of only a single message, so the limitation on the size of
-        individual messages is only reduced by the overhead of the batch format. This similarly affects the
-        producer's <code>batch.size</code> configuration.</li>
+        Previously the setting applied to batches of compressed messages, or to non-compressed messages individually.
+        A message batch may consist of only a single message, so in most cases, the limitation on the size of
+        individual messages is only reduced by the overhead of the batch format. However, there are some subtle implications
+        for message format conversion (see <a href="#upgrade_11_message_format">below</a> for more detail). This similarly affects
+        the producer's <code>batch.size</code> configuration.</li>
```

Comment:
Also, is it worth mentioning that the consumer and replication max bytes and max partition bytes limits don't apply for the first record batch instead of the first message? Or is that too obscure?